js-intro.js

```diff
@@ -39,18 +52,48 @@ if (imLost || imFrustrated) {
 
 // 5. Update the imLost and imFrustated variables above the statement to the values required to log "Keep coding!"
 
+//done
+
 // 6a. Recall the mantra variable from the first question. Write the code that logs each letter of mantra using a for loop.
 
+var mantra = "Be the dev"
+for(let i=0; i<mantra.length; i++){
+    console.log(mantra[i])
+}
+
 // Stretch: Using a while loop.
 
+let i=0
+while(i<mantra.length){
+    console.log(mantra[i])
+    i++
+}
+
 // Super Stretch: Using forEach().
 
 // Consider the following variable:
 
-var message = "thisisateststring"
+var testString = "thisisateststring"
+var newTestString = 12345
 
 // 7. Write a function that takes a string like message as an argument and returns the string without vowels.
 
+
+const noVowels = (words) =>{
+    if(words != "string"){
```

Comment:
Nice job adding the additional filter of taking in the correct data type, but you need to use a typeof method to get the correct evaluation of "string".